.github/workflows/ci.yml

```diff
@@ -129,14 +129,14 @@ jobs:
           args: --all-features --no-deps --workspace
 
       - uses: aws-actions/configure-aws-credentials@v4.0.1
-        if: github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name
+        if: ( github.event_name == 'merge_group' || github.event_name == 'push' ) || github.repository == github.event.pull_request.head.repo.full_name
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-west-1
 
       - name: Upload to S3
-        if: github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name
+        if: ( github.event_name == 'merge_group' || github.event_name == 'push' ) || github.repository == github.event.pull_request.head.repo.full_name
```

Comment:
Is "or" correct here? Will this publish the report twice, or is the event not a "push" from a merge group?

Reply:
Correct, the event from `merge_group` is called `check_requested`, so this runs once when the merge queue is setup and then again on the final push to `main`.